_docs-sources/iac/overview/index.md

```diff
@@ -4,24 +4,18 @@ The Gruntwork Infrastructure as Code Library (IaC Library) is a collection of re
 
 ## Modules
 
-Modules are reusable code components that are used to deploy and manage specific pieces of infrastructure. These modules encapsulate the configuration and resource definitions required to create and manage a particular component, such as a VPC, ECS cluster, or an Auto Scaling Group. For more information on modules check out the [Modules page](/iac/overview/modules/).
+Modules are "infrastructure building blocks" authored by Gruntwork and written in Terraform. They capture a singular best-practice pattern for specific pieces of infrastructure and are designed to be both limited in scope and highly reusable. They typically represent one part of a use case you want to accomplish. For example, the `vpc-flow-logs` module does not create a VPC, it only adds the VPC Flow Logs functionality to an existing VPC. To learn more, check out [What is a module?](/iac/overview/modules/).
```

Comment:
We've been removing all usage of `check out` to `refer to` because it's less casual

```suggestion
Modules are "infrastructure building blocks" authored by Gruntwork and written in Terraform. They capture a singular best-practice pattern for specific pieces of infrastructure and are designed to be both limited in scope and highly reusable. They typically represent one part of a use case you want to accomplish. For example, the `vpc-flow-logs` module does not create a VPC, it only adds the VPC Flow Logs functionality to an existing VPC. To learn more, refer to [What is a module?](/iac/overview/modules/).
```

Reply:
Sounds good!